tsconfig.esm.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+ "compilerOptions": {
+    "baseUrl": "src",
+    "outDir": "dist/esm",
+    "rootDir": "src",
+    "target": "es5",
+    "module": "es6",
+    "tsBuildInfoFile": "node_modules/.cache/uniforms.esm.tsbuildinfo"
```

Comment:
Same fix, please change the name to `uniforms-patternfly` instead of `uniforms`.